package.json

```diff
@@ -744,7 +744,7 @@
     "devDependencies": {
         "@types/mocha": "^9.0.0",
         "@types/node": "^10.17.17",
-        "@types/vscode": "^1.76.0",
+        "@types/vscode": "^1.74.0",
         "tslint": "^5.20.1",
```

Comment:
Lower version??

Reply:
I am reviewing the history of external dependencies, and it looks like this is a ticking 🧨 .

When dependencies are eventually mass updated, won't the breakage caused by 1.76 return, resulting in a regression? What can be done to guard against this expected future breakage? Surely hope alone is not enough.

Based on the merged `yarn.lock` it appears [that version 1.76.0 was resolved as a correct alternative to 1.74.0](https://github.com/microsoft/vscode-makefile-tools/commit/5b4d318518d4bb2a54ec90af791b904ef079d9bc#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de), and that is what's being used for the release package:

```diff
diff --git a/yarn.lock b/yarn.lock
index 2b7784e..f32eedc 100644
--- a/yarn.lock
+++ b/yarn.lock
@@ -231,7 +231,7 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-10.17.60.tgz#35f3d6213daed95da7f0f73e75bcc6980e90597b"
   integrity sha512-F0KIgDJfy2nA3zMLmWGKxcH2ZVEtCZXHHdOQs2gSaQ27+lNeEfGxzkIw90aXswATX7AZ33tahPbzy6KAfUreVw==

-"@types/vscode@^1.76.0":
+"@types/vscode@^1.74.0":
   version "1.76.0"
   resolved "https://registry.yarnpkg.com/@types/vscode/-/vscode-1.76.0.tgz#967c0fbe09921818bbf201f1cbcb81b981c6249c"
   integrity sha512-CQcY3+Fe5hNewHnOEAVYj4dd1do/QHliXaknAEYSXx2KEHUzFibDZSKptCon+HPgK55xx20pR+PBJjf0MomnBA==
```